data/categories.txt

```diff
@@ -4185,6 +4185,46 @@ sk:3Kino
 sw:4Sinema|burudani
 fa:سینما
 
+leisure-bowling_alley|@entertainment
+en:4Bowling Alley
+ar:ﺞﻨﻴﻟﻮﺑ ﺔﻟﺎﺻ
+be:Боўлінг
+bg:Боулинг писта
+ca:Bolera
+cs:Bowlingová dráha
+da:Bowling bane
+de:Bowlingbahn
+el:Αίθουσα σφαιρίσεως
+es:Bolera
+et:Bowlingusaal
+eu:Bolatokia
+fa:ﮓﻨﯿﻟﻮﺑ ﻦﻟﺎﺳ
+fi:Keilarata
+fr:Allée de bowling
+he:גנילואב םלוא
+hu:Tekepálya
+id:Arena Bowling
+it:Sala da bowling
+ja:ボーリング場
+ko:볼링장
+mr:गोलंदाजी गल्ली
+nb:Bowlinghall
+nl:Bowlingbaan
+pl:Kręgielnia
+pt:Pista de boliche
```

Comment:
pt:Pista de bowling

Reply:
Should `pt-BR:Pista de boliche` be changed to bowling as well?

Reply:
No, just PT. PT-BR is already correct (Pista de boliche), thanks!

Reply:
cool, thank you for the translations :)